GCRCatalogs/buzzard.py

```diff
@@ -175,11 +175,23 @@ def _subclass_init(self,
                 'velocity_z': 'truth/VZ',
             }
 
+            if self.version > 1.6:
+
+                for i, b in enumerate(['Z_vista', 'Y_vista', 'J_vista', 'H_vista', 'Ks_vista', 'u_lsst', 
+                                       'g_lsst', 'r_lsst', 'i_lsst', 'z_lsst', 'y_lsst', 'Y_wfirst',
+                                       'J_wfirst', 'H_wfirst', 'K_wfirst']):
+
+                    self._quantity_modifiers['Mag_true_{}_z0'.format(b)] = (_abs_mask_func, 'auxmag/AMAG/{}'.format(i))
+                    self._quantity_modifiers['mag_true_{}'.format(b)] = (_mask_func, 'auxmag/TMAG/{}'.format(i))
+                    self._quantity_modifiers['mag_{}'.format(b)] = (_mask_func, 'auxmag/LMAG/{}'.format(i))
+
+            else:
+                self._quantity_modifiers['Mag_true_{}_lsst_z0'.format(b)] = (_abs_mask_func, 'lsst/AMAG/{}'.format(i))
+                self._quantity_modifiers['mag_true_{}_lsst'.format(b)] = (_mask_func, 'lsst/TMAG/{}'.format(i))
+
             for i, b in enumerate('ugrizyY'):
```

Comment:
Just want to make sure it is intended that this for loop should be outside of the else clause?